docs/user_guide.md

```diff
@@ -1296,9 +1296,9 @@ optional arguments:
   --filter FILTER       Filter results. The filter string has the following
                         format:
                         [<SEVERITIES>]:[<CHECKER_NAMES>]:[<FILE_PATHS>]:
-                        [<DETECTION_STATUSES>]:[<REVIEW_STATUSES>] where
+                        [<DETECTION_STATUS>]:[<REVIEW_STATUS>] where
```

Comment:
If you change the form of the word from plural to singular, change it everywhere: `SEVERITIES`, `CHECKER_NAMES`, `FILE_PATHS`.